src/Views/email_activate_show.php

```diff
@@ -19,9 +19,10 @@
                 <?= csrf_field() ?>
 
                 <!-- Code -->
-                <div class="mb-2">
-                    <input type="text" class="form-control" name="token" placeholder="000000" inputmode="numeric"
+                <div class="form-floating mb-2">
+                    <input type="text" class="form-control" id="floatingTokenInput" name="token" placeholder="000000" inputmode="numeric"
                         pattern="[0-9]*" autocomplete="one-time-code" value="<?= old('token') ?>" required />
+                    <label for="floatingTokenInput">Token</label>
```

Comment:
Shield supports several different languages.

Reply:
🤦🏽‍♂️. Thanks for the reminder.